flutter/javascript/src/index.js

```diff
@@ -0,0 +1,64 @@
+const wdio = require('webdriverio');
+const assert = require('assert');
+const find = require('appium-flutter-finder');
+
+const osSpecificOps =
+  process.env.APPIUM_OS === 'android'
+    ? {
+        platformName: 'Android',
+        platformVersion: '*',
+        deviceName: '*',
+        udid: '',
+        app: 'https://kobiton-devvn.s3-ap-southeast-1.amazonaws.com/apps-test/demo/app-debug-all.apk',
+        fullReset: true,
+        noReset: false,
+        newCommandTimeout: 360
+      }
+    : process.env.APPIUM_OS === 'ios'
+    ? {
+        platformName: 'iOS',
+        platformVersion: '*',
+        deviceName: '*',
+        udid: '',
+        noReset: true,
+        app: 'https://kobiton-devvn.s3-ap-southeast-1.amazonaws.com/apps-test/demo/Runner.zip',
+        newCommandTimeout: 360,
+        waitForQuiescence: false,
+        autoAcceptAlerts: true
+      }
+    : {};
+
+const opts = {
+  protocol: 'https',
+  hostname: 'username:apiKey@api.kobiton.com',
```

Comment:
Define const for username & token

`const username = process.env.KOBITON_USERNAME || ''
const apiKey = process.env.KOBITON_API_KEY || ''
const deviceName = process.env.KOBITON_DEVICE_NAME || ''

const kobitonServerConfig = {
  protocol: 'https',
  host: 'api.kobiton.com',
  auth: `${username}:${apiKey}`
}`

Reply:
I fixed. Thanks!